jupyter_core/application.py

```diff
@@ -203,8 +203,9 @@ def load_config_file(self, suppress_errors=True):
         except ConfigFileNotFound:
             self.log.debug("Config file not found, skipping: %s", config_file_name)
         except Exception:
-            # For testing purposes.
-            if not suppress_errors:
+            # Reraise errors for testing purposes, or if set in
+            # self.raise_config_file_errors
+            if not suppress_errors or self.raise_config_file_errors:
```

Comment:
Can we put `not suppress_errors` in parentheses? I never remember operator precedence orders.